lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -243,7 +243,7 @@ def test_query_counts_cached(self, store_type, with_storage_backing):
             self._get_blocks(
                 course,
                 expected_mongo_queries=0,
-                expected_sql_queries=14 if with_storage_backing else 13,
+                expected_sql_queries=13 if with_storage_backing else 12,
```

Comment:
These query reductions are because edx-when will now early-exit before querying any waffle flags, since the experiments waffle flag requires a request object. And tests that don't set that up will thus cause edx-when to make one less query.